typescript/simple-mercari-web/src/App.css

```
@@ -15,35 +17,84 @@
 
 .Listing {
   background-color: #222427;
-  min-height: 8vh;
+  padding: 16px;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: left;
-  font-size: calc(10px + 1vmin);
+  justify-content: center;
+  font-size: 16px;
   color: white;
 }
 
-.ItemList {
-  background-color: #222427;
-  min-height: 8vh;
+form {
+  width: 100%;
+  max-width: 800px;
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 50px;
   align-items: center;
-  justify-content: left;
-  font-size: calc(10px + 1vmin);
-  color: white;
+  justify-content: center;
+}
+
+input {
+  padding: 10px;
+  border-radius: 5px;
+  font-size: 14px;
+  background-color: white;
+  color: black;
+}
+
+.name,
+.text,
+.file {
+  flex: 1;
+  min-width: 150px;
+}
+
+.button {
+  padding: 10px 16px;
+  background-color: white;
+  color: black;
+  font-size: 14px;
+  cursor: pointer;
 }
 
-.App-link {
-  color: #61dafb;
+.button:hover {
+  background-color: #0056b3;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+.ItemList_div {
```

NIT: If you'd like, you can also do the following:

```suggestion
.ItemList > div {
```

---

I fixed.